### ISSUE TITLE
Middle click for new Window

### ISSUE THREAD

**Issue by Irame** [sadesyllas/vscode-workspace-switcher#42]
I would love to have the ability to use middle click on a workspace in the list to open it in a new window, just like the O-shaped icon.

**Comment by wc-davide** [comment]
I agree... or maybe `ctrl+click` so it's similar to web pages... but either way this would be a really handy feature

**Comment by Minimistro32** [comment]
@wc-davide, middle click and ctrl+click do the same thing on web pages (open hyperlinks in a new window).
